### PR TITLE
chore: updated dependencies, included alternative commandline start o…

### PR DESCRIPTION
**Pull request by eschrewe**
## Description
- Updated Spring-Boot Dependency to version 3.1.4
- Updated Snakeyaml to version 2.2
- Included workaround for launching the backend application from command line

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -180,7 +175,7 @@
     <resources>
       <!-- add basic application properties -->
       <resource>
-          <directory>${project.basedir}/src/main/resources</directory>
+          <directory>backend/src/main/resources</directory>
```
Comment (tom-rm-meyer-ISST) [inline backend/pom.xml:178]: please undo as the variable resolves to the backend folder.

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Small change in pom needed. Rest is great. Please also comment & close respective dependabot PRs if applicable in future (I already did it here :))

File: backend/pom.xml (hunk 1 above)
Comment (eschrewe) [reply backend/pom.xml]: Done!

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM.

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM

**Pull request merged**